cashctrl_api/client.py

```diff
@@ -43,42 +43,28 @@ def __init__(self, organisation: str = None, api_key: str = None):
     # ----------------------------------------------------------------------
     # API Requests
 
-    def request(
-        self, method: str, endpoint: str, data: dict = None, params: dict = None
-    ) -> Response:
-        """Send an API request to CashCtrl.
+    def request_with_retry(self, method: str, url: str, **kwargs) -> Response:
```

Comment:
Would it be useful to add parameter called `waitTime` to use in the `time.sleep(waitTime)` instead of `time.sleep(1)`, and make it default to be a `1`?